README.md

```diff
@@ -444,76 +441,14 @@ Use this parameter if you do not want a confirmation prompt to interrupt automat
 
 ## Contributing
 
-For guidelines on contributing to the project, please refer to our [Contributing](docs/CONTRIBUTING.md) page. 
+Please read our [contribution guide](./CONTRIBUTING.md).
 
 ## Release Notes
 
-### v3.0.0
-
-- Serverless 3 compatibility
-- Updated dependencies
-
-### v2.8.0
-
-- Adds a lot of patches and fixes from old deps to new ones.
-
-### v2.7.6
-
-- Adds support for easier NPM deploys via GitHub releases
-- Allows to set the "redirect" property of routing rules to be an empty string, which is a legitimate value. - [EnricoPicci](https://github.com/EnricoPicci)
-- Clearer details on where the plugin uploads files when key prefixed. [Lilja](https://github.com/Lilja)
-- Why 2.7.6 you ask? Because there were a lot of CI tests that made this release happen. A awful lot of them.
-
```

Comment:
Everything below here is not on the releases page https://github.com/fernando-mc/serverless-finch/releases

There are no release notes for 2.6.0 and below

Should we maybe leave 2.60 and below release notes in the read me and have the link to the releases tab for others ?